coap-netty/src/main/java/org/opencoap/coap/netty/NettyCoapTransport.java

```diff
@@ -51,7 +52,24 @@ public NettyCoapTransport(Bootstrap bootstrap, Function<DatagramPacket, Transpor
 
     @Override
     public void start() {
-        init(bootstrap.bind().syncUninterruptibly().channel());
+        if (bootstrap.config().remoteAddress() != null) {
+            init(bootstrap.connect().syncUninterruptibly().channel());
+        } else {
+            init(bootstrap.bind().syncUninterruptibly().channel());
+
+            // for random port update bootstrap with actual port but only if SO_REUSEPORT is enabled
+            if (bindToRandomPort() && reusePortIsEnabled()) {
+                bootstrap.localAddress(channel.localAddress());
```

Comment:
Does it work so, that after the first server using this bootstrap is started, then all the following server using the same bootstrap will use the updated version of it (with the port properly set)?

Reply:
Yes, exactly that way.